base/bucket_gocb_test.go

```diff
@@ -473,6 +474,44 @@ func TestIncrCounter(t *testing.T) {
 
 }
 
+// TestIncrAmtZero covers the special handling in Incr for when amt=0 on an unknown key
+func TestIncrAmtZero(t *testing.T) {
+
+	if UnitTestUrlIsWalrus() {
+		t.Skip("This test only works against Couchbase Server")
+	}
```

Comment:
What part of this test isn't walrus-compatible?  It looks like everything is just going through the bucket interface, so it would be preferable to ensure walrus is compatible, even if that requires a walrus enhancement.

Reply:
Walrus has different Incr behaviour based on the presence of a non-zero default (seen in CBG-357).

I could change the test to supply zero as `def` instead and run on both?

Reply:
Given the discussion we had about SG usages of Incr, I'll update this to run on all buckets with `amt=0, def=0` and other `def` testing can be done as part of that other CBG.